news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,138 @@
+
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/effective-miss-count.jpg)
+
+### Changes to Difficulty Performance Points Values
+
+The following changes should result in impressive scores on harder beatmaps rewarding a significant increase of performance point values, with top player [syaron105](https://osu.ppy.sh/users/8741695) gaining over 6400 total PP!
+
+Please note that these changes only affect the difficulty part of performance, which refers to the Star Rating value under a mathematically scaled formula.
+
+- A further EZ multiplier of **0.985x** has been added, further supplementing its nerf mentioned above.
+- A HR multiplier of **1.05x** has been added, which was added as a way to rebalance the changes found within accuracy PP.
+- Difficulty pp now decreases by the square of decreasing accuracy instead of linearly, which applies a huge nerf to previously overweighted low-accuracy scores.
+
+The new accuracy-scaling formula is mapped here.
+  
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/difficulty-accuracy-scaling.jpg)
+  
+### Changes to Accuracy Performance Points Values
+  
+While Difficulty PP is straightforward, Accuracy performance contained a plethora of issues, both to do with reading-based mods (HD, FL) and mods that affect the Great Hit Window (EZ, DT, HR)
+
+- The impact of the size of the Great (300) hit window on accuracy pp has been reduced significantly. This makes PP less dependent on the OD value of a map, which greatly reduces the issues of low SR maps, with a tight hit window.
+- Star rating now affects accuracy pp, which replaces most of the scaling from the previous impact of the great hit window. This further supplements the nerfs the accuracy weighting on low-SR scores and buffs high-SR scores and thus significantly limits abuse of low-SR 3-mod/4-mod scores.
+- An additional HDFL bonus has been added to accuracy PP. As this mod combination requires total memorisation of maps, the bonus scales with map length, rewarding longer maps more.
+
+The new hidden-flashlight bonus is graphed here.
+
+![](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/accuracy-mod-bonus.jpg)
+  
+## Next steps
+  
+Future changes will involve rhythm, which hasn't been touched, and reading, an entirely new concept to osu!taiko, as well as many other large and small-scale changes. For people eager to dive into the technicalities, WIP details are available [here](https://docs.google.com/document/d/1Z5GC4DMqOVzeIERMSK3qpQaqjq-sVnhbuoxAwy9qxDs/edit). If you would like to give feedback or help contribute, please join the [osu!dev Discord server](https://discord.gg/ppy) and direct your comments towards the `#difficulty-taiko` channel.
```

Comment:
Wouldn't it be a good idea to link pp dev server here as well, or is it not something that should be in the newspost?